server/src/cm_job_task.cpp

```diff
@@ -5206,12 +5232,43 @@ ts_loaddb (nvplist *req, nvplist *res, char *_dbmt_error)
       argv[argc++] = "--" LOAD_IGNORE_CLASS_L;
       argv[argc++] = ignore_class_file;
     }
+  if (uStringEqual (no_user_specified_name, "yes"))
+    {
+      argv[argc++] = "--" LOAD_NO_USER_SPECIFIED_NAME_L;
+    }
+  if (schema_file_list != NULL && !uStringEqual (schema_file_list, "none"))
+    {
+      schema_file_list_opt_flag = true;
+      snprintf (schema_file_list_opt, PATH_MAX, "%s%s", "--" LOAD_SCHEMA_FILE_LIST_L "=", schema_file_list);
+      argv[argc++] = schema_file_list_opt;
+    }
   argv[argc++] = dbname;
   argv[argc++] = NULL;
 
   make_temp_filepath (cubrid_err_file, sco.dbmt_tmp_dir, "loaddb_err_tmp", TS_LOADDB, PATH_MAX);
 
-  retval = run_child (argv, 1, NULL, tmpfile, cubrid_err_file, NULL);    /* loaddb */
+  if (schema_file_list_opt_flag)
+    {
+      char loaddb_exe_path[PATH_MAX];
+      char drive [_MAX_DRIVE];
```

Comment:
The variables (drive, dir) are used only in window. How about moving the variable under #if defined(WINDOWS)?

Reply:
Thank you for your comment!
I will change that.